test/test_fcntl.rs

```diff
@@ -49,15 +49,48 @@ fn test_readlink() {
 mod linux_android {
     use std::io::prelude::*;
     use std::os::unix::prelude::*;
+    use std::io::SeekFrom;
 
     use libc::loff_t;
 
-    use nix::fcntl::{SpliceFFlags, FallocateFlags, fallocate, splice, tee, vmsplice};
+    use nix::fcntl::{SpliceFFlags, FallocateFlags, fallocate, splice, tee, vmsplice, copy_file_range};
     use nix::sys::uio::IoVec;
     use nix::unistd::{close, pipe, read, write};
 
     use tempfile::{tempfile, NamedTempFile};
 
+    /// This test creates a temporary file containing the contents
+    /// 'foobarbaz' and uses the `copy_file_range` call to transfer
+    /// 3 bytes at offset 3 (`bar`) to another empty file at offset 0. The
+    /// resulting file is read and should contain the contents `bar`.
+    /// The from_offset should be updated by the call to reflect
+    /// the 3 bytes read (6).
+    ///
+    /// FIXME: test is disabled for linux based builds, because Jenkins
```

Comment:
s/Jenkins/Travis/
This limitation sucks.  I'll do a quick PR to see if we can do better by switching Linux builds to Cirrus.